src/scripts/main.js

```diff
@@ -1,7 +1,75 @@
-'use strict';
+const Game = require('../modules/Game.class');
+
+// eslint-disable-next-line no-unused-expressions
+('use strict');
 
 // Uncomment the next lines to use your game instance in the browser
 // const Game = require('../modules/Game.class');
 // const game = new Game();
 
 // Write your code here
+document.addEventListener('DOMContentLoaded', () => {
+  const game = new Game();
+
+  const startButton = document.getElementById('start-button');
+  const scoreElement = document.getElementById('score');
+  const statusElement = document.getElementById('status');
+  const gameBoard = document.getElementById('game-board');
+
+  function updateUI() {
+    const state = game.getState();
+    const cells = gameBoard.getElementsByClassName('field-cell');
```

Comment:
It's generally a good practice to cache DOM elements that you'll need to reference multiple times. Consider storing the 'cells' collection in a variable outside of the 'updateUI' function. This way, you won't need to query the DOM every time 'updateUI' is called, which can improve performance.